app/firstmodule/controllers.py

```diff
@@ -0,0 +1,70 @@
+from flask import request, redirect, render_template, Blueprint
+from sqlalchemy import func
+
+from app import db
+from app.firstmodule.models import HoroscopeModel
+
+module = Blueprint('horoscope', __name__, url_prefix='/')
+
+
+@module.route('/create-horoscope', methods=['GET', 'POST'])
```

Comment:
According to common sense of REST API create endpoint doesn't need to contain 'create' prefix. It should be implicitly clear from the method POST.
Don't mix views (endpoints for rendering) and API endpoints. So these two ones should be defined at least in separate modules.